### ISSUE TITLE
Does the SVL support simulation of robot arms, e.g. UR10?

### ISSUE THREAD

**Issue by manone888** [lgsvl/simulator#1468]
Hi guys, your webpage https://www.svlsimulator.com/use-cases/#robotics mentions Robotics simulation. I'd like to know if I could control a robot arm, e.g. UR10 via ROS in the SVL simulator. This will require sending commands from ROS nodes to the simulator and sending joints' positions from the simulator to ROS nodes.

Thank you. I look forward to your reply.

**Comment by EricBoiseLGSVL** [comment]
Yes, we are working on the next release that supports articulated bodies.

**Comment by manone888** [comment]
Hi Eric, thank you for your reply. 

It sounds great! Do you have an estimated time for the next release?

**Comment by EricBoiseLGSVL** [comment]
Estimated end of this month

**Comment by manone888** [comment]
Wow, that would be so great! I really look forward to it.

Thanks a lot.

**Comment by ShawnSchaerer** [comment]
Any updates on when this will be supported?

**Comment by EricBoiseLGSVL** [comment]
@ShawnSchaerer testing the release all this week :)

**Comment by ShawnSchaerer** [comment]
> @ShawnSchaerer testing the release all this week :)

Great, thanks!
